test/fixtures/empty/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "test",
+  "version": "1.0.0"
+}
```

Comment:
This needed to be added because otherwise the logic in `framework-info` would detect the root project's `package.json`. This ensures that we're truly testing when no framework is detected in a project